### PR TITLE
JMAP news posted by Tuan LE CONG

### PR DESCRIPTION
**Pull request by yvesdaxmaz**
Jmap Barcamp 2019

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "_includes"]
 	path = _includes
-	url = https://github.com/jmapio/jmap.git
+	url = https://github.com/yvesdaxmaz/jmap.git
```
Comment (marclaporte) [inline .gitmodules]: Really?

File: news.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ permalink: /news.html
 title: JMAP News
 ---
 # JMAP News
+
+## Tuan LE CONG
```
Comment (neilj) [inline news.md]: The title should be the title of the article: "Jmap Barcamp 2019"

File: .gitmodules (hunk 1 above)
Comment (neilj) [reply .gitmodules]: Yes, this needs to be removed.

**Review by neilj — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (yvesdaxmaz) [reply .gitmodules]: I have update the .gitmodules

**Pull request merged**